versions/1/manifest.spec.yml

```diff
@@ -170,27 +170,26 @@ spec:
               examples:
                 - ">=7.9.0"
     input_groups:
-      description: List of policy groups offered by this package.
+      description: List of input groups offered by this package.
       type: array
       items:
         type: object
         additionalProperties: false
         properties:
           name:
-            description: Name of policy group.
+            description: Name of input group.
             type: string
             examples:
-              - Logs
-              - Metrics
+              - logs
+              - metrics
```

Comment:
I think we want to treat the `name` property like an ID since it will be referenced from within `policy_templates`. So I think lowercasing makes sense here.